### ISSUE TITLE
Feature Request: Include IMM constants from imm.h in win32con

### ISSUE THREAD

**Issue by songyuc** [mhammond/pywin32#2441]
Hello,

I'm working on a Python application that needs to control the Input Method Editor (IME) on Windows. While `win32con` includes IME-related window messages like `WM_IME_CONTROL`, it doesn't include the IMM constants defined in `imm.h`.

Currently, we need to manually define these constants:
```python
IMC_SETCONVERSIONMODE = 0x0002
IMC_SETOPENSTATUS = 0x0006
IME_CMODE_ALPHANUMERIC = 0x0000
```

These constants are essential for IME operations. Here's a typical use case where we need these constants:
```python
def set_input_method_to_english():
    ime_hwnd = get_ime_window()
    if not ime_hwnd:
        return False
    # Using WM_IME_CONTROL (available in win32con)
    # but IMC_SETOPENSTATUS needs to be manually defined
    ctypes.windll.user32.SendMessageW(ime_hwnd, win32con.WM_IME_CONTROL, 
                                     IMC_SETOPENSTATUS, 0)
    return True
```

Would it be possible to include these IMM-related constants from `imm.h`? This would:
1. Provide a more complete set of IME-related constants
2. Eliminate the need for manual constant definitions
3. Ensure consistency with Windows SDK values
4. Make IME-related development in Python more convenient

The relevant constants include but are not limited to:
- IMC_GETCONVERSIONMODE (0x0001)
- IMC_SETCONVERSIONMODE (0x0002)
- IMC_GETSENTENCEMODE (0x0003)
- IMC_SETSENTENCEMODE (0x0004)
- IMC_GETOPENSTATUS (0x0005)
- IMC_SETOPENSTATUS (0x0006)

Thank you for considering this request!

**Comment by Avasam** [comment]
Hi. Are you sure these constants still exist in `imm.h` ? 
Internationalization messages seem to be for an EOL microsoft product (Windows Embedded?) as per this documentation: https://learn.microsoft.com/en-us/previous-versions/windows/embedded/ms921492(v=msdn.10)
![image](https://github.com/user-attachments/assets/8ceee8f8-7a17-41cc-801d-36bcd746882c)

And I cannot find them in my local installation (SDK version `10.0.19041.0`, `10.0.22000.0`, or `10.0.22621.0`):
![image](https://github.com/user-attachments/assets/1b48143c-d222-46f2-88f5-612f5126e3b3)

The constants could be manually added, diverging further from the autogeneration of constants from existing headers. But even the link above only documents their names, not their values. So to be able to include them all, I'd like to know more about your installation (like which version of the SDKs) and how you found these constants in `imm.h`
> the IMM constants defined in `imm.h`
> Would it be possible to include these IMM-related constants from `imm.h` in `win32con`?
> The relevant constants include but are not limited to

**Comment by songyuc** [comment]
Dear @Avasam,

I hope this message finds you well. I wanted to take a moment to thank you for your guidance and support regarding my feature request. Your insights were incredibly helpful and led me to revisit my code and the `win32con` module more carefully.

Upon further investigation, I realized that I had mistakenly confused the numerical values with the corresponding constant names. The constants I actually needed are:
```
IMN_SETCONVERSIONMODE
IMN_OPENSTATUSWINDOW
```

I discovered that these constants are already included in the `win32con` module, which means my original issue has been resolved. Your advice prompted me to double-check my assumptions, and I am grateful for that.

Thank you again for your patience and for pointing me in the right direction. Your assistance has been invaluable, and I truly appreciate the time and effort you took to help me.

Best regards,  
Yucheng Song
